### PR TITLE
Add Vagabond Profession

### PR DESCRIPTION
**Pull request by iamlcarus**
`SUMMARY: Content "Added Vagabond profession."`

Simply adds a Vagabond profession.
I felt dissatisfied with the Hobo profession since it seems more to be a jokey/stereotypical representation of a homeless person(they literally start with a bindle), rather than an example of a capable survivor. After all, if homeless people are used to living without many of life's amenities, wouldn't they be more well-suited to the apocalypse than most other survivors?
Sorry if there's something I did wrong with the formatting, my first time ever making a PR.

### CODE REVIEW

File: data/json/professions.json [hunk 1]
```diff
@@ -136,7 +136,26 @@
       { "item": "cookbook_human", "bonus": [ [ "CANNIBAL" ] ] },
       { "item": "teleumbrella", "bonus": [ [ "ALBINO" ] ] }
     ]
-  },
+  },{
+    "type": "profession",
+    "ident": "vagabond",
+    "name": "Vagabond",
+    "description": "Circumstances left you wandering, with no home, no family, no friends. But the world you knew is gone, and maybe your experiences relying on yourself to survive could be useful in this new one.",
```
Comment (Night-Pryanik) [inline data/json/professions.json]:
```suggestion
    "description": "Circumstances left you wandering, with no home, no family, no friends.  But the world you knew is gone, and maybe your experiences relying on yourself to survive could be useful in this new one.",
```

File: data/json/professions.json [hunk 2]
```diff
@@ -136,7 +136,26 @@
...
+    "points": 3,
+    "skills": [ 
+        { "level": 2, "name": "unarmed" },
+        { "level": 2, "name": "survival"},
```
Comment (Night-Pryanik) [inline data/json/professions.json]:
```suggestion
        { "level": 2, "name": "survival" },
```

File: data/json/professions.json [hunk 3]
```diff
@@ -136,7 +136,26 @@
...
+        { "level": 2, "name": "tailor"},
```
Comment (Night-Pryanik) [inline data/json/professions.json]:
```suggestion
        { "level": 2, "name": "tailor" },
```

File: data/json/professions.json [hunk 4]
```diff
@@ -136,7 +136,26 @@
...
+        { "level": 2, "name": "fabrication"},
```
Comment (Night-Pryanik) [inline data/json/professions.json]:
```suggestion
        { "level": 2, "name": "fabrication" },
```

File: data/json/professions.json [hunk 5]
```diff
@@ -136,7 +136,26 @@
...
+        { "level": 2, "name": "cooking"}
```
Comment (Night-Pryanik) [inline data/json/professions.json]:
```suggestion
        { "level": 2, "name": "cooking" }
```

**Comment by Night-Pryanik** [comment]
Please use linter (http://dev.narc.ro/cataclysm/format.html) to properly format your changes.

**Comment by iamlcarus** [comment]
> Please use linter (http://dev.narc.ro/cataclysm/format.html) to properly format your changes.
Thank you, it should be properly formatted now.

**Comment by Night-Pryanik** [comment]
Also please add a summary line for our auto-changelog-maker which should look like `SUMMARY: Content "Added Vagabond profession."`

**Comment by I-am-Erk** [comment]
I think you've way overstated the skillset of the average homeless wanderer... You're going from overly negative stereotypes in the Hobo profession over the other side to overly positive ones. Most people living on the street have a basic blue collar labour sort of skillset. Probably fabrication 2, cooking 1, tailoring 1 perhaps, unarmed 1. You've given them a pretty broad set at a pretty good starting level, and without the negatives thrown in its hard to see what would have led to this person living on the street.

I loved the idea brought up in discord of giving this profession a randomized negative trait from a list, using the same trait groups code as NPC classes used.

File: data/json/professions.json [hunk 6]
```diff
@@ -136,8 +136,43 @@
...
+        "jeans",
+        "tshirt",
+        "gloves_light",
+        "hat_ball",
+        "duffelbag",
+        "long_underpants",
+        "boots",
+        "socks_wool",
+        "socks",
+        "long_undertop",
```
Comment (I-am-Erk) [inline data/json/professions.json]: Long underwear bottoms are pretty common, but tops are a bit more specialized. I don't think they're as easy to find at thrift stores. Probably best left out.

File: data/json/professions.json [hunk 7]
```diff
@@ -136,8 +136,43 @@
...
+        "hoodie",
+        "folding_poncho_on",
+        "knit_scarf",
```
Comment (I-am-Erk) [inline data/json/professions.json]: Patchwork scarf might be more thematic

File: data/json/professions.json [hunk 8]
```diff
@@ -136,8 +136,43 @@
...
+      { "level": 2, "name": "fabrication" },
+      { "level": 2, "name": "cooking" }
+    ],
+    "items": {
+      "both": [
+        "jeans",
+        "tshirt",
+        "gloves_light",
+        "hat_ball",
+        "duffelbag",
```
Comment (I-am-Erk) [inline data/json/professions.json]: You might up that torso encumbrance and the thematic feel by also giving them a backpack.

File: data/json/professions.json [hunk 9]
```diff
@@ -136,8 +136,43 @@
...
-  {
+  },{
+    "type": "profession",
+    "ident": "vagabond",
+    "name": "Vagabond",
+    "description": "Circumstances left you wandering, with no home, no family, no friends.  But the world you knew is gone, and maybe your experiences relying on yourself to survive could be useful in this new one.",
+    "points": 3,
+    "skills": [
+      { "level": 2, "name": "unarmed" },
+      { "level": 2, "name": "survival" },
```
Comment (I-am-Erk) [inline data/json/professions.json]: The kind of stuff covered in survival, eg skinning or foraging, just don't fit with the urban street dweller imo

**Comment by Night-Pryanik** [comment]
Jenkins rebuild

File: data/json/professions.json (hunk 7 above)
Comment (iamlcarus) [reply data/json/professions.json]: I can't find the patchwork scarf

File: data/json/professions.json [hunk 10]
```diff
@@ -136,8 +136,42 @@
       { "item": "cookbook_human", "bonus": [ [ "CANNIBAL" ] ] },
       { "item": "teleumbrella", "bonus": [ [ "ALBINO" ] ] }
     ]
-  },
-  {
+  },{
```
Comment (ZhilkinSerg) [inline data/json/professions.json]:
```suggestion
   },
   {
```

File: data/json/professions.json [hunk 11]
```diff
@@ -136,8 +136,42 @@
...
+        "knit_scarf",
+        "jug_plastic",
+        "can_beans",
+        "pockknife",
+        "matches"
+      ],
+      "male": [ "boxer_briefs" ],
+      "female": [ "bra", "panties" ]
+    }
+  },{
```
Comment (ZhilkinSerg) [inline data/json/professions.json]:
```suggestion
  },
  {
```

**Pull request merged**